### PR TITLE
add doc for tiflash encryption-at-rest

### PR DESCRIPTION
**Pull request by lidezhu**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: encryption-at-rest.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ TiKV 从 v4.0.0 起支持静态加密，即在 [CTR](https://zh.wikipedia.org/wi
 当前版本 (v4.0.0) 的 TiKV 加密功能还存在待改善之处，将在未来版本中进行改善。
 
 * TiDB 集群部署后，大部分用户数据都存储在 TiKV 节点上。这部分数据可使用加密功能进行加密。但有少量元数据存储在 PD 节点上（例如用作 TiKV Region 边界的二级索引键）。截至 v4.0.0，PD 尚未支持静态加密功能。建议使用存储级加密（例如文件系统加密）来保护存储在 PD 中的敏感数据。
-* 截至 v4.0.0，TiFlash 尚未支持静态加密功能。TiKV 与 TiFlash 一起部署时，存储在 TiFlash 中的数据不会被加密。
+* TiFlash 从 v4.0.5 开始支持静态加密功能，详情参阅 [TiFlash 静态加密](/encryption-at-rest.md#TiFlash 静态加密)。TiKV 与 v4.0.5 之前版本的 TiFlash 一起部署时，存储在 TiFlash 中的数据不会被加密。
```
Comment (TomShawn) [inline encryption-at-rest.md:20]:
```suggestion
* TiFlash 从 v4.0.5 开始支持静态加密功能，详情参阅 [TiFlash 静态加密](/encryption-at-rest.md#tiflash-静态加密-从-v405-版本开始引入)。TiKV 与 v4.0.5 之前版本的 TiFlash 一起部署时，存储在 TiFlash 中的数据不会被加密。
```

File: encryption-at-rest.md [hunk 2]
```diff
@@ -126,3 +126,7 @@ region = "us-west-2"
 ```
 ./br restore full --pd <pd-address> --storage "s3://<bucket>/<prefix> --s3.region <region>"
 ```
+
+## TiFlash 静态加密 <span class="version-mark">从 v4.0.5 版本开始引入</span>
+
+TiFlash 从 v4.0.5 起支持静态加密。数据密钥由 TiFlash 生成，TiFlash（包括 TiFlash Proxy）写入的所有文件，包括数据文件，Schema 文件，临时文件等，均由当前数据密钥加密。TiFlash 支持的加密算法，加密配置方法（配置项在 `tiflash-learner.toml` 中）和监控项含义（如果 TiFlash 中部署了 Grafana 组件，可以查看 **TiFlash-Proxy-Details** -> **Encryption**）等均与 TiKV 一致。
```
Comment (TomShawn) [inline encryption-at-rest.md:132]:
```suggestion
TiFlash 从 v4.0.5 起支持静态加密。数据密钥由 TiFlash 生成。TiFlash（包括 TiFlash Proxy）写入的所有文件，包括数据文件、Schema 文件、临时文件等，均由当前数据密钥加密。TiFlash 支持的加密算法、加密配置方法（配置项在 `tiflash-learner.toml` 中）和监控项含义等均与 TiKV 一致。

如果 TiFlash 中部署了 Grafana 组件，可以查看 **TiFlash-Proxy-Details** -> **Encryption**。
```

File: enable-tls-between-components.md [hunk 3]
```diff
@@ -162,7 +162,7 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
     cdc server --pd=https://127.0.0.1:2379 --log-file=ticdc.log --addr=0.0.0.0:8301 --advertise-addr=127.0.0.1:8301 --ca=/path/to/ca.pem --cert=/path/to/ticdc-cert.pem --key=/path/to/ticdc-key.pem --cert-allowed-cn="client1,client2"
     ```
 
-- TiFlash
+- TiFlash <span class="version-mark">从 v4.0.5 版本开始引入</span>
 
     在 `config` 文件或命令行参数中设置：
```
Comment (JaySon-Huang) [inline enable-tls-between-components.md:167]: 我们目前能在命令行参数中设置相关参数吗？

Comment (lidezhu) [reply enable-tls-between-components.md:167]: 看起来应该不行，可以找 @hanfei1991 确认下。

**Review by zanmato1984 — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@lidezhu @zanmato1984 @JaySon-Huang Can wen merge this PR now?

**Comment by JaySon-Huang** [comment]
@TomShawn Yes, we can merge this PR.

File: enable-tls-between-components.md [hunk 4]
```diff
@@ -66,8 +66,8 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
         key-path = "/path/to/pd-server-key.pem"
         ```
 
-    - TiFlash
-    
+    - TiFlash <span class="version-mark">从 v4.0.5 版本开始引入</span>
```
Comment (TomShawn) [inline enable-tls-between-components.md:69]:
```suggestion
    - TiFlash（从 v4.0.5 版本开始引入）
```

File: enable-tls-between-components.md [hunk 5]
```diff
@@ -162,7 +162,7 @@ aliases: ['/docs-cn/dev/enable-tls-between-components/','/docs-cn/dev/how-to/sec
     cdc server --pd=https://127.0.0.1:2379 --log-file=ticdc.log --addr=0.0.0.0:8301 --advertise-addr=127.0.0.1:8301 --ca=/path/to/ca.pem --cert=/path/to/ticdc-cert.pem --key=/path/to/ticdc-key.pem --cert-allowed-cn="client1,client2"
     ```
 
-- TiFlash
+- TiFlash <span class="version-mark">从 v4.0.5 版本开始引入</span>
```
Comment (TomShawn) [inline enable-tls-between-components.md:165]:
```suggestion
- TiFlash（从 v4.0.5 版本开始引入）
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**